torch/_subclasses/fake_tensor.py

```diff
@@ -60,6 +61,17 @@ def __torch_dispatch__(cls, func, types, args=(), kwargs=None):
 
         # Run the original computation
 
+        # _to_copy fails when run with FakeTensors to cuda device
+        # TODO: debug
```

Comment:
You're talking about the meta implementation here right? It makes sense, since we can't actually produce the cuda data (but for fake tensor that's OK, because the fake cuda device doesn't have data anyway)